cmd/common.go

```diff
@@ -80,7 +80,11 @@ func syncMain(filenames []string, dry bool, parallelism int) error {
 		return err
 	}
 	// prepare to read the current state from Kong
-	config.Workspace = targetContent.Workspace
+	if len(workspace) > 0 {
```

Comment:
The more idiomatic way to do this `workspace != ""`.

Reply:
Please also add a warning (using print.DeletePrintln from print package in this repo) when the workspace supplied via cli and that supplied via state file do not match up.

Reply:
Done.